lib/section_reslugger.rb

```diff
@@ -109,10 +109,8 @@ def change_note
   end
 
   def publish_manual
-    observers = ManualObserversRegistry.new
     service = PublishManualService.new(
       manual_repository: RepositoryRegistry.create.manual_repository,
-      listeners: observers.publication,
```

Comment:
Sorry @chrisroos I just noticed I'd missed this in 2fd96ea98! I also have a commit on my current working branch that fixes this, but I'll get rid of that.